assets/global.js

```diff
@@ -819,6 +820,35 @@ class VariantSelects extends HTMLElement {
     });
   }
 
+  updateVariantStatuses() {
+    const typeOfInput = this.id.includes('variant-radios') ? 'input[type=radio]' : 'option';
+    //Grab all the existing variants based on the selected option1
+    const selectedOptionOneVariants = JSON.parse(this.querySelector('[type="application/json"]').textContent).filter(variant => this.querySelector(`${typeOfInput}:checked`).value === variant.option1);
+    //Array of all the input wrappers we have (option1, option2 and option3)
+    const inputWrappers = [...this.querySelectorAll('.product-form__input')];
+    //Loop through each one except option1 and compare the input values to the
+    inputWrappers.forEach((option, index) => {
+      if (index === 0) return;
+      //Select the inputs
+      const optionInputs = [...option.querySelectorAll(typeOfInput)]
```

Comment:
I wonder if I should actually be more specific in case people edit their theme to add more customization options 🤔 Ill go with that second option for now 👍

Reply:
Can you any think of any example of customization that would result in having `value` attributes inside variant inputs? I tried to, but couldn't haha.

Either way, the second option "kinda" violates the principle of keeping variant radios logic outside this component, but I think it's fine because it's just an extra selector.